src/Data/Bool/Properties.agda

```diff
@@ -762,7 +762,21 @@ proof (T? false) = ofⁿ λ()
 T?-diag : ∀ b → T b → True (T? b)
 T?-diag true  _ = _
 
-push-function-into-if : ∀ (f : A → B) x {y z} →
-                        f (if x then y else z) ≡ (if x then f y else f z)
-push-function-into-if _ true  = refl
-push-function-into-if _ false = refl
+if-float : ∀ (f : A → B) b {xᵗ xᶠ} →
```

Comment:
I'm not a huge fan of the change of `x` and `y` for `xᵗ` and `xᶠ`, as its against the style-guide. Can we switch it back?